sceptre/cli/list.py

```diff
@@ -127,3 +126,39 @@ def list_change_sets(ctx, path, url):
 
     for response in responses:
         write(response, context.output_format)
+
+
+@list_group.group(name="local")
```

Comment:
A `local` command doesn't really make sense to me and seems unnecessary.  I think i prefer just `sceptre list stacks prod/mystack.yaml`.   if we decide to support getting attributes from remote templates in the future then i think we can just add a `--remote` flag?

Reply:
Reverted.